packages/jest-worker/src/__performance_tests__/test.js

```diff
@@ -20,7 +20,7 @@ assert(process.argv[3], 'Pass the number of iterations');
 
 const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
 const method = process.argv[2];
-const calls = +process.argv[3];
+const calls = Number(process.argv[3]);
```

Comment:
keep in mind that Number, Boolean and String can be overriden with something faulty, so `+`, `!!` and `"" + 'string'` are mostly safest approach
but I guess in jest repo it doesn't matter much